File: app/src/main/java/com/cs407/calculatorapp/ResultActivity.java
package com.cs407.calculatorapp;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class ResultActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_result);

        Intent intent = getIntent();
        String result = intent.getStringExtra("result");

        TextView resultTextView = findViewById(R.id.resultTextView);
        resultTextView.setText("Result: " + result);
    }
}

File: app/src/main/java/com/cs407/calculatorapp/ResultActivity.kt
package com.example.calculatorapp

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.cs407.calculatorapp.R

class ResultActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_result)

        val result = intent.getStringExtra("result")

        val resultTextView = findViewById<TextView>(R.id.resultTextView)
        resultTextView.text = result
    }
}
